File: src/practica-3-2/holamundo.c
#include <stdio.h>

int main() {
    char nombre[50];
    printf("¿Cuál es tu nombre? ");
    scanf("%s", nombre);
    printf("Hola %s. Este programa está hecho en el lenguaje de programación: C.\n", nombre);
    return 0;
}

File: src/practica-3-2/holamundo.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("¿Cuál es tu nombre? ");
        String nombre = scanner.nextLine();
        System.out.println("Hola " + nombre + ". Este programa está hecho en el lenguaje de programación: Java.");
    }
}

